package.json

```diff
@@ -125,10 +127,19 @@
     "start:prod": "REACT_APP_ENV=prod node scripts/start.js",
     "build": "node scripts/build.js",
     "test": "node scripts/test.js --env=jsdom",
-    "lint:fix": "eslint --fix --",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
     "extract-messages": "NODE_ENV=production babel ./src  --out-file /dev/null",
     "manage:translations": "NODE_ENV=production ./node_modules/.bin/babel-node scripts/translator.js",
-    "build:langs": "npm run extract-messages && npm run manage:translations"
+    "build:langs": "npm run extract-messages && npm run manage:translations",
+    "precommit": "lint-staged",
+    "prepush": "npm run lint:fix"
+  },
+  "lint-staged": {
+    "*.js": [
```

Comment:
does this follow the same rules as `.eslintignore`?

Reply:
it will not, but the `npm run lint:fix` will